resolwe_bio/tools/make_report.py

```diff
@@ -66,6 +128,33 @@ def parse_targetPCRmetrics(metrics_report):
         return dict(zip(labels, values))
 
 
+def get_mean_coverage(covd):
+    """Calculate mean coverage from coverageBed (.covd) report file."""
+    with open(covd) as pbc:
+        cov = 0
+        lines = 0
+        for line in pbc:
+            cov = cov + int(line.strip().split('\t')[6])
+            lines = lines + 1
+
+    return float(cov) / lines
```

Comment:
You don't need to cast ints to floats, if you add `from __future__ import division`, as this is default behaviour in python 3. Transition to python 3 will also be easier, if you use `future` features.